Cargo.lock

```diff
@@ -3183,6 +3205,16 @@ dependencies = [
  "unicode-width 0.1.4 (registry+https://github.com/rust-lang/crates.io-index)",
 ]
 
+[[package]]
+name = "thread-id"
```

Comment:
This crate doesn't compile for wasm32-unknown-unknown, but since it's behind a feature flag it doesn't really matter.

Reply:
I'm not whether the `Cargo.lock` should be updated, since this is under a feature flag. Should I revert the changes to `Cargo.lock`?

Reply:
No, the `Cargo.lock` always considers all options I think. So it's correct to update it.